### PR TITLE
Add recipe for emmix version 1.3

### PR DESCRIPTION
**Pull request by gregvonkuster**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/emmix/1.3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - certifi
+    - gcc # [linux]
+
+test:
+  # EMMIX is interactive (requiring user input), so cannot be tested.
+
+about:
+  home: 'https://people.smp.uq.edu.au/GeoffMcLachlan/emmix/emmix.html'
+  summary: 'A tool that fits a mixture model of multivariate normal or t-distributed components to a given data set.'
+  license: 'Available freely for non-commercial use only'
```
Comment (bgruening) [inline recipes/emmix/1.3/meta.yaml]: be warned here, this is hard to integrate into Galaxy and distribute

File: recipes/emmix/1.3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  md5: 66523b6873e820bde7561182dd522898
+
+build:
+  skip: True  #[osx]
+  number: 0
+
+requirements:
+  build:
+    - certifi
+    - gcc # [linux]
```
Comment (bgruening) [inline recipes/emmix/1.3/meta.yaml]: libgcc missing as run-dep?

File: recipes/emmix/1.3/meta.yaml (hunk 1 above)
Comment (gregvonkuster) [inline recipes/emmix/1.3/meta.yaml]: Thanks for the info about this.  This EMMIX package will be a dependency of the soon coming PlantTribes Galaxy tools which will be installable from the Tool Shed.  How much of a problem will this be?

Comment (bgruening) [inline recipes/emmix/1.3/meta.yaml]: Let's put is this way, in theory you need somehow to ensure that your users know about this restriction ...

File: recipes/emmix/1.3/meta.yaml (hunk 2 above)
Comment (gregvonkuster) [inline recipes/emmix/1.3/meta.yaml]: Thanks for the reminder @bgruening !

**Comment by gregvonkuster** [comment]
Thanks for adding the wget dependency I missed (sorry about that) @bgruening.  I'm not sure why it is still failing :(

**Comment by bgruening** [comment]
Because it is called gnu-wget, silly me. Sorry.

**Comment by gregvonkuster** [comment]
Thanks a bunch @bgruening I'm always in your debt!  ;)

**Pull request merged**